src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -30,7 +32,8 @@ internal static class LogRecordExtensions
         this OtlpCollector.ExportLogsServiceRequest request,
         SdkLimitOptions sdkLimitOptions,
         OtlpResource.Resource processResource,
-        in Batch<LogRecord> logRecordBatch)
+        in Batch<LogRecord> logRecordBatch,
+        ExperimentalFeatures experimentalFeatures)
```

Comment:
nit: Personally I would make this the second parameter to the method (instead of the last) so that all settings/options logically come before the other things.